docs/kyma-environment-broker/04-01-cls-architecture.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Cloud Logging Service Architecture
+type: Architecture
+---
+
+Cloud Logging Service (CLS) is a managed logging service for shipping logs from Runtimes. It uses the Elastic Stack (open source logging platform Elasticsearch, Logstash, Kibana) to store, parse, and visualize the application log data coming from Kubernetes applications.
```

Comment:
```suggestion
The Cloud Logging service (CLS) is a managed logging service for shipping logs from various runtimes. It uses the Elastic Stack (open source logging platform Elasticsearch, Logstash, Kibana) to store, parse, and visualize the application log data coming from Kubernetes applications.
```

Reply:
Official [CLS docu](https://pages.github.tools.sap/perfx/cloud-logging-service/consumption/getting-started/) writes it this way.

I think "runtimes" is a normal word, no need to capitalize.